### DISCUSSION TITLE
WooCommerce PDF Invoice

### DISCUSSION THREAD

**Discussion by Uhtredtb** [inspireui/support#5253] [New Feature/Ideas Requests] [answered]
It would be great to support WooCommerce PDF Invoice and users can view it on the order history tab. 

Please kindly consider this and integrate with this plugin 
https://codecanyon.net/item/woocommerce-pdf-invoice/5951088

**Comment by shubhamrane48** [comment]
I thing below plugins are more suitable in Fluxstore applications. One more thing is that this plugins are developed but WooCommerce.

Plugins-

1. https://woocommerce.com/products/pdf-invoices/
2. https://wordpress.org/plugins/woocommerce-pdf-invoices-packing-slips/

**Reply by Uhtredtb** [reply]
But this one bring more, feature to the PDF invoice. 

It can add product image, custom string and more.

Please check: https://codecanyon.net/item/woocommerce-pdf-invoice/5951088

Would be great if it can be added to this plugin too?
@shubhamrane48

**Comment by minhcasi** [comment, marked as answer]
#5241

**Comment by tryabp** [comment]
Yes, invoice feature is must in fluxstore. I m sure they will add it in future updates.
